### PR TITLE
Add notes about Copter Tailsitter frame types

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-future-wiki-changes.rst [hunk 1]
```diff
@@ -471,11 +471,25 @@ For landing in fixed wing, manual throttle controlled modes, there is an ``RCx_O
 
 On Tailsitters page, under Vectored Thrust:
 -------------------------------------------
-
+gitk
```
Comment (kd0aij) [inline common/source/docs/common-future-wiki-changes.rst:474]: unintentional?

**Review by kd0aij — CHANGES_REQUESTED** [review]
LGTM except for comment

File: common/source/docs/common-future-wiki-changes.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-future-wiki-changes.rst:474]: the joys of automatic keyboard and mouse switching between machines

Comment (kd0aij) [reply common/source/docs/common-future-wiki-changes.rst:474]: :) At least it didn't go out as an email
I've started doing "git diff" before every commit... also helps to catch unintentional whitespace changes

Comment (Hwurzburg) [reply common/source/docs/common-future-wiki-changes.rst]: added notes about motor mask and q_options for copter TS without control surfaces...

**Comment by Hwurzburg** [comment]
@IamPete1 can you approve?

**Review by IamPete1 — APPROVED** [review]
LGTM, thanks Henry

**Pull request merged**